src/openhab.cpp

```diff
@@ -224,63 +211,63 @@ void OpenHAB::connect() {
             _flagOpenHabConnected = false;
             disconnect();
             qCDebug(m_logCategory) << "openhab not reachable";
+        } else if (reply->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt() == 200) {
+            qCDebug(m_logCategory) << reply;
+            qCDebug(m_logCategory) << reply->header(QNetworkRequest::ContentTypeHeader).toString();
+            qCDebug(m_logCategory) << reply->header(QNetworkRequest::LastModifiedHeader).toDateTime().toString();;
+            qCDebug(m_logCategory) << reply->header(QNetworkRequest::ContentLengthHeader).toULongLong();
+            qCDebug(m_logCategory) << reply->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt();
+            qCDebug(m_logCategory) << reply->attribute(QNetworkRequest::HttpReasonPhraseAttribute).toString();
+
+
+            _sseReconnectTimer->setSingleShot(true);
+            _sseReconnectTimer->setInterval(2000);
+            _sseReconnectTimer->stop();
+
+            QObject::connect(&_sseNetworkManager, &QNetworkAccessManager::finished, this, &OpenHAB::streamFinished);
+            QObject::connect(_sseReconnectTimer, &QTimer::timeout, this, &OpenHAB::onSseTimeout);
+
+            startSse();
+            getItems(true);
+            setState(CONNECTED);
         }
+        QObject::disconnect(&_nam, &QNetworkAccessManager::finished, this, 0);
     });
-    _checkProcessOpenHabAvailability.start("curl", QStringList() << "-s" << _url);
```

Comment:
Thank you for removing this ugly hack! :+1:

Reply:
:)